src/api/events/events-controller.js

```diff
@@ -0,0 +1,204 @@
+const EventType = require('./event-type');
```

Comment:
I'll need to dig deeper into the functionality of that class.
Does it work with RTS devices too?